emails/views.py

```diff
@@ -867,6 +896,39 @@ def _handle_bounce(message_json):
     return HttpResponse("OK", status=200)
 
 
+def _handle_complaint(data: ComplaintMessage) -> HttpResponse:
+    complaint_type = data.complaint.complaintFeedbackType
+    if data.mail.commonHeaders and data.mail.commonHeaders.replyTo:
+        reply_to = ",".join(data.mail.commonHeaders.replyTo)
+    else:
+        reply_to = None
+    extra = {
+        "complaint_type": complaint_type.value if complaint_type else None,
+        "sender": data.mail.source,
+        "recipients": ",".join(
+            [cr.emailAddress for cr in data.complaint.complainedRecipients]
+        ),
+        "message_id": data.mail.messageId,
+        "complaint_subtype": data.complaint.complaintSubType,
+        "feedback_id": data.complaint.feedbackId,
+        "user_agent": data.complaint.userAgent,
+        "reply_to": reply_to,
+    }
+    info_logger.info(f"Complaint {data.channelType.value} received.", extra=extra)
```

Comment:
Since these logs are for specific SES/SNS operations maybe we should consider a separate logger for just email handling starting with handling complains and delivery. Wdyt?